ocaml/xapi/xapi_xenops.ml

```diff
@@ -28,6 +28,20 @@ module Rrdd = Rrd_client.Client
 open Xenops_interface
 open Xapi_xenops_queue
 
+let assert_final_power_state_is' ~__context ~self ~expected =
```

Comment:
I did this deliberately to have a distinction between `Xapi_vm_lifecycle.assert_final_power_state_is` (also used) and the new function. But I can find a different name.